core/src/main/python/wlsdeploy/tool/discover/topology_discoverer.py

```diff
@@ -69,59 +69,68 @@ def discover(self):
+            model_folder_name, folder_result = self._get_log_filters()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
 
-        model_folder_name, folder_result = self._get_virtual_hosts()
-        discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+            model_folder_name, folder_result = self._get_reliable_delivery_policies()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
 
-        model_folder_name, folder_result = self._get_xml_entity_caches()
-        discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+            model_folder_name, folder_result = self._get_virtual_hosts()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
 
-        model_folder_name, folder_result = self._get_xml_registries()
-        discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+            model_folder_name, folder_result = self._get_xml_entity_caches()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
 
-        model_folder_name, folder_result = self.get_managed_executor_template()
-        discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+            model_folder_name, folder_result = self._get_xml_registries()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
 
-        model_folder_name, folder_result = self.get_managed_thread_factory_template()
-        discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+            model_folder_name, folder_result = self.get_managed_executor_template()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
 
-        model_folder_name, folder_result = self.get_managed_scheduled_executor_service()
-        discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+            model_folder_name, folder_result = self.get_managed_thread_factory_template()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
 
-        model_folder_name, folder_result = self._get_ws_securities()
-        discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+            model_folder_name, folder_result = self.get_managed_scheduled_executor_service()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+
+            model_folder_name, folder_result = self._get_ws_securities()
+            discoverer.add_to_model_if_not_empty(self._dictionary, model_folder_name, folder_result)
+        finally:
+            if current_tree is not None:
+                current_tree()
```

Comment:
Should this be wlst_helper.current_tree()

Reply:
No, it should not